doc/radosgw/adminops.rst

```diff
@@ -273,13 +273,15 @@ TBD.
 Get User Info
 =============
 
-Get user information.
+Get user information. Cap ``users`` must be set to ``read`` to run this operation.
+If additional cap ``users-without-keys`` is set to ``read`` or ``*``, S3 Key and Swift keys will not be
+included in the response unless the user running this operation is the system user.
```

Comment:
i didn't expect `users-without-keys` to *remove* permissions this way. no other caps work like this

i thought the admin was going to give out `users-without-keys` caps *instead* of `users` when they shouldn't have access to keys

so `RGWOp_User_Info::check_caps()` would grant access if the user had either `users` or `users-without-keys` read caps, but omit keys for the latter

Reply:
That was the other way I was thinking of approaching it. The only conflict that would occur is if both `users` and `users-without-keys` were set (on accident) and the read bit was set on each too.

In that case do you agree that the effect of the `users-without-keys` cap would prevail on a `get user info` call?

Reply:
permissions should be additive, so if `users=read` grants read access to everything, then `users=read`+`users-without-keys=read` should too